### PR TITLE
MTM-48188: announcement of nimbus-jwt usage in sso

### PR DESCRIPTION
**Pull request by chbakar**

### CODE REVIEW

**Review by Michal-Bak — APPROVED** [review]

File: content/release-10-17-0/announcements-10-17-0.md [hunk 1]
```diff
@@ -17,6 +17,16 @@ layout: bundle
 
 #### Implemented
 
+##### Support for new token type "at+jwt" in SSO
```
Comment (BeateRixen) [inline content/release-10-17-0/announcements-10-17-0.md:20]:
```suggestion
##### Support for new token type in single sign-on authentication
```

File: content/release-10-17-0/announcements-10-17-0.md [hunk 2]
```diff
@@ -17,6 +17,16 @@ layout: bundle
...
+
+Cumulocity supports new standard OAUTH2 tokens with "at+jwt" type Header Parameter. For more information, see https://datatracker.ietf.org/doc/html/rfc9068#name-header.
```
Comment (BeateRixen) [inline content/release-10-17-0/announcements-10-17-0.md:22]:
```suggestion
Cumulocity IoT supports new standard OAUTH2 tokens with the "at+jwt" type header parameter. For more information on this parameter, see [https://datatracker.ietf.org/doc/html/rfc9068#name-header](https://datatracker.ietf.org/doc/html/rfc9068#name-header).
```

File: content/release-10-17-0/announcements-10-17-0.md [hunk 3]
```diff
@@ -17,6 +17,16 @@ layout: bundle
...
+Single-sign-on users managed by new authorization servers (i.e, generating JWT token with type "at+jwt") can thus login successfully into Cumulocity.
```
Comment (BeateRixen) [inline content/release-10-17-0/announcements-10-17-0.md:23]:
```suggestion
Single-sign-on users managed by new authorization servers (that is, generating a JWT token with type "at+jwt") can thus login successfully into Cumulocity IoT.
```

File: content/release-10-17-0/announcements-10-17-0.md [hunk 4]
```diff
@@ -17,6 +17,16 @@ layout: bundle
...
+To process this new token type, Cumulocity uses the [Nimbus JOSE + JWT](https://connect2id.com/products/nimbus-jose-jwt), which can be enabled by the tenants with the
```
Comment (BeateRixen) [inline content/release-10-17-0/announcements-10-17-0.md:24]:
```suggestion
To process this new token type, Cumulocity IoT uses the [Nimbus JOSE + JWT](https://connect2id.com/products/nimbus-jose-jwt), which can be enabled by the tenants with the
```

**Review by BeateRixen — APPROVED** [review]

**Comment by BeateRixen** [comment]
@chbakar Should I merge this PR?

**Comment by chbakar** [comment]
> @chbakar Should I merge this PR?

@BeateRixen I am awaiting @sarangovindarajan approval here.

File: content/release-10-17-0/announcements-10-17-0.md [hunk 5]
```diff
@@ -17,6 +17,11 @@ layout: bundle
 
 #### Implemented
 
+##### Support for new token type in single sign-on authentication
+
+Cumulocity IoT supports new standard OAUTH2 tokens with the "at+jwt" type header parameter. For more information on this parameter, see [https://datatracker.ietf.org/doc/html/rfc9068#name-header](https://datatracker.ietf.org/doc/html/rfc9068#name-header).
```
Comment (sarangovindarajan) [inline content/release-10-17-0/announcements-10-17-0.md:22]: Let us call it OAuth2 access tokens.

File: content/release-10-17-0/announcements-10-17-0.md [hunk 6]
```diff
@@ -17,6 +17,11 @@ layout: bundle
...
+Single-sign-on users managed by new authorization servers (that is, generating a JWT token with type "at+jwt") can thus login successfully into Cumulocity IoT.
```
Comment (sarangovindarajan) [inline content/release-10-17-0/announcements-10-17-0.md:23]: Same as above. Let us call it OAuth2 access tokens.

**Review by sarangovindarajan — APPROVED** [review]

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**